lib/ProductOpener/Numbers.pm

```diff
@@ -130,7 +130,12 @@ sub convert_string_to_number ($value) {
 	}
 	# Remove remaining commas that can be used as separators
 	$value =~ s/,//g;
-	$value += 0;
+
+	# If number has been recorded as 0.0 without decimals then want to preserve this
```

Comment:
Today we don't have a way to store "4.0" or "0.0", they are converted to 4 and 0. The API expects numbers for both reading and writing (even though we do support strings for writing).